src/Native/Windows/build-native.cmd

```diff
@@ -10,36 +10,53 @@ set __IntermediatesDir=""
 set __BuildArch=AnyCPU
 set __VCBuildArch=x86_amd64
 set CMAKE_BUILD_TYPE=Release
+set __EnvironmentSet=false
+set __VSVersion=vs2013
```

Comment:
Ideally, we should be keeping these things similar where possible between the different dotnet repos. Both coreclr and corert have `set __VSVersion=vs2015`.